src/website/src/pages/ec/components/hero-banner/docs/usage.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A Hero banner displays a prominent message and related optional actions.
+  Banners are displayed on the homepage below the site header of EC sites. It
+  conveys an important message to your audience. Hero banners are a good way to
+  channel your visitors to complete a specific task (ex: register to an event,
+  apply for funding... ). The user can choose to ignore them or interact with
+  them at any time.
+</Paragraph>
+
+## Anatomy
+
+Banners should have a focused message and provide users with clear actions to take. They should be placed at the top of a layout or screen. When scrolling, banners typically move with content and scroll off the screen. Only one banner should be shown at a time.
```

Comment:
```suggestion
Banners should have a focused message and provide users with clear actions to take. They should be placed at the top of a layout or screen. When scrolling, banners typically move with content and scroll off the screen. Only one banner should be shown at a time.
```
Using the combination of a photo, or illustration, with copy and “call to action” button. It engage user as the centerpiece in the page. At the same time, not all elements listed here are required. It depends on the message we want to communicate with our users. 
```

Reply:
I don't understand